src/pages/ProductDetail/component/Recommend.js

```diff
@@ -0,0 +1,20 @@
+import './Recommend.scss';
+
+const Recommend = ({ product }) => {
+  return (
```

Comment:
```suggestion
const Recommend = ({ product }) => {
  const {id, img_url, img_alt, detail_img_url, name, price} = product;
  
  return (
```
개인적으로는 한번 더 구조분해 할당을 해서 유지보수와 가독성에 도움이 될 수 있도록
구성하는 것이 더 깔끔해보입니다!